docs/coderunner.md

```diff
@@ -0,0 +1,37 @@
+## Code Runner Support (currently Linux and MacOS only)
```

Comment:
I'm pretty sure Code Runner works on Windows too. Could you look into making a set of instructions for windows as well?

Reply:
Added Windows code